PhysicsTools/NanoAOD/python/custom_jme_cff.py

```diff
@@ -18,118 +17,126 @@
 
 import copy
 
-bTagCSVV2    = ['pfDeepCSVJetTags:probb','pfDeepCSVJetTags:probbb','pfDeepCSVJetTags:probc']
-bTagDeepCSV  = ['pfCombinedInclusiveSecondaryVertexV2BJetTags']
+bTagCSVV2    = ['pfCombinedInclusiveSecondaryVertexV2BJetTags']
+bTagDeepCSV  = ['pfDeepCSVJetTags:probb','pfDeepCSVJetTags:probbb','pfDeepCSVJetTags:probc']
 bTagDeepJet  = [
   'pfDeepFlavourJetTags:probb','pfDeepFlavourJetTags:probbb','pfDeepFlavourJetTags:problepb',
   'pfDeepFlavourJetTags:probc','pfDeepFlavourJetTags:probuds','pfDeepFlavourJetTags:probg'
 ]
 from RecoBTag.ONNXRuntime.pfParticleNetAK4_cff import _pfParticleNetAK4JetTagsAll
 bTagDiscriminatorsForAK4 = bTagCSVV2+bTagDeepCSV+bTagDeepJet+_pfParticleNetAK4JetTagsAll
 
+from RecoBTag.ONNXRuntime.pfDeepBoostedJet_cff import _pfDeepBoostedJetTagsAll
+from RecoBTag.ONNXRuntime.pfParticleNet_cff import _pfParticleNetJetTagsAll
+
+btagHbb = ['pfBoostedDoubleSecondaryVertexAK8BJetTags']
+btagDDX = [
+  'pfDeepDoubleBvLJetTags:probHbb',
+  'pfDeepDoubleCvLJetTags:probHcc',
+  'pfDeepDoubleCvBJetTags:probHcc',
+  'pfMassIndependentDeepDoubleBvLJetTags:probHbb',
+  'pfMassIndependentDeepDoubleCvLJetTags:probHcc',
+  'pfMassIndependentDeepDoubleCvBJetTags:probHcc'
+]
+btagDDXV2 = [
+  'pfMassIndependentDeepDoubleBvLV2JetTags:probHbb',
+  'pfMassIndependentDeepDoubleCvLV2JetTags:probHcc',
+  'pfMassIndependentDeepDoubleCvBV2JetTags:probHcc'
+]
+
 #
 # By default, these collections are saved in NanoAODs:
-# - ak4gen (GenJet in NanoAOD), slimmedGenJets in MiniAOD 
-# - ak8gen (GenJetAK8 in NanoAOD), slimmedGenJetsAK8 in MiniAOD 
-# Below is a list of genjets that we can save in NanoAOD. Set 
+# - ak4gen (GenJet in NanoAOD), slimmedGenJets in MiniAOD
+# - ak8gen (GenJetAK8 in NanoAOD), slimmedGenJetsAK8 in MiniAOD
+# Below is a list of genjets that we can save in NanoAOD. Set
 # "enabled" to true if you want to store the jet collection
 config_genjets = [
-  { 
-    "jet"     : "ak8gen",    
-    "enabled" : False, 
-  },  
+  {
+    "jet"     : "ak6gen",
```

Comment:
What's the motivation for the change from ak8 -> ak6 in the naming ?

Reply:
I had set the new AK8 GEN jet collection reclustering in a function [here](https://github.com/cms-sw/cmssw/blob/6e83ab4ee2d62986883fb4871cf4961ba420bd83/PhysicsTools/NanoAOD/python/custom_jme_cff.py#L952). The change in the line above is just to put a dummy example using ak6gen. The ak6gen jet clustering is switched off.